src/test/regress/expected/binary_protocol.out

```diff
@@ -25,6 +25,43 @@ SELECT * FROM t ORDER BY id;
  10
 (10 rows)
 
+EXPLAIN (ANALYZE TRUE, TIMING FALSE, COSTS FALSE, SUMMARY FALSE) SELECT id FROM t ORDER BY 1;
+                            QUERY PLAN
+---------------------------------------------------------------------
+ Sort (actual rows=10 loops=1)
+   Sort Key: remote_scan.id
+   Sort Method: quicksort  Memory: 25kB
+   ->  Custom Scan (Citus Adaptive) (actual rows=10 loops=1)
+         Task Count: 2
+         Data received: 11 bytes
```

Comment:
Just asking, but if it is `2GB`, will it show `2GB` or `2147483648 bytes`? If the latter it might be good to make it more readable.

Reply:
I copied the way of generating from how @pykello did it in his PR: https://github.com/citusdata/citus/pull/3864/files#diff-44a470ebd7842955b493ca913941c03dR288-R289

So I assumed it would be good. However, reading the source code of the `ExplainPropertyInteger` function,  it turns out that it does simply suffix the "bytes" string though. So, I agree that's not great.

I'll take a look on how to change that.

Reply:
I used `DirectFunctionCall1(pg_size_pretty, ...)` now to get a nicely formatted string instead. I used the same for "Intermediate Data Size" section added by @pykello.